modules/UI/UI.js

```diff
@@ -1267,7 +1269,11 @@ UI.showDeviceErrorDialog = function (micError, cameraError) {
 
     message = `${message}${doNotShowWarningAgainSection}`;
 
-    messageHandler.openDialog(
+    // To make sure we don't have multiple error dialogs open at the same time,
+    // we will just close the previous one if we are going to show a new one.
+    deviceErrorDialog && deviceErrorDialog.close();
+
+    deviceErrorDialog = messageHandler.openDialog(
```

Comment:
Both dialogs will report message about gUM error. And this error might happen when user switches between devices. So if he tried to switch from first device to second and got an error, and then tried switch from first device to a third one and got an error, I don't see any problems in automatically hiding the previous gUM error notification as it's not relevant anymore.
The fact is that I don't know even if such situation is currently possible (after several fixes in lib-jitsi-meet), so this more like an additional check to be sure.